src/neptune/internal/operation_processors/operation_storage.py

```diff
@@ -34,11 +34,12 @@ def __init__(self, data_path: Union[str, Path]):
             self._data_path = data_path
         else:
             self._data_path = Path(data_path)
-
         # initialize directories
         os.makedirs(self.data_path, exist_ok=True)
         os.makedirs(self.upload_path, exist_ok=True)
 
+        self._data_path = self._data_path.resolve()
```

Comment:
In general, I don't like this "expanding" pattern with reassigning to the value. Let's use `= Path(data_path).resolve()`